pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -71,6 +71,24 @@ func (h *HeartBeat) heartBeat(heartBeatInterval time.Duration, stopCh chan struc
 	// 1 minute with a 1.2 jitter + the time it takes for the heartbeat function to run (sliding == true).
 	// So the amount of time between heartbeats randomly varies between 1min and 2min12sec + the heartbeat function execution time.
 	wait.JitterUntil(h.do, heartBeatInterval, 1.2, true, stopCh)
+	now, err := json.Marshal(metav1.Now())
+	if err != nil {
+		log.DefaultLogger().Reason(err).Errorf("Can't determine date")
+		return
+	}
+	cpuManagerEnabled := false
+	kubevirtSchedulable := "false"
+	var data []byte
+	data = []byte(fmt.Sprintf(`{"metadata": { "labels": {"%s": "%s", "%s": "%t"}, "annotations": {"%s": %s}}}`,
+		v1.NodeSchedulable, kubevirtSchedulable,
+		v1.CPUManager, cpuManagerEnabled,
```

Comment:
This is confusing, I would have expected the variables to be named `kubevirtNonSchedulable` and `cpuManagerDisabled`...
But I don't think we even need those variables, it probably makes more sense to hardcode `false` and `"false"` here.

Reply:
Done!